pallets/vc-management/Cargo.toml

```diff
@@ -26,25 +26,25 @@ pallet-teebag = { path = "../teebag", default-features = false }
 
 # benchmarking
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42", default-features = false, optional = true }
-test-utils = { path = "../test-utils", default-features = false, optional = true }
 
 [dev-dependencies]
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42" }
 pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42" }
-pallet-group = { path = "../../pallets/group" }
 pallet-timestamp = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42" }
 sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.42" }
-test-utils = { path = "../test-utils" }
+pallet-group = { path = "../../pallets/group" }
+pallet-teebag = { path = "../teebag", features = ["test-util"] }
+
 
 [features]
 default = ["std"]
 runtime-benchmarks = [
-    "frame-benchmarking/runtime-benchmarks",
+    "frame-benchmarking?/runtime-benchmarks",
```

Comment:
is it typo or specific syntax?

Reply:
Thanks Igor, it's not a typo - but it's wrong :)

We should have enabled this by default - I'll fix it